pkg/v1/tkg/client/package_helper.go

```diff
@@ -38,6 +38,9 @@ func GetCorePackagesFromClusterBootstrap(clusterBootstrap *runtanzuv1alpha3.Clus
 	// Core packages CNI, CSI and CPI (other than Kapp) installed in tkg-system namespace in case of tkgm, and in case of tkgs installed in vmware-system-tkg namespace
 	if clusterBootstrap.Spec.CNI != nil && clusterBootstrap.Spec.CNI.ValuesFrom != nil && clusterBootstrap.Spec.CNI.ValuesFrom.ProviderRef != nil {
 		name := strings.TrimSuffix(clusterBootstrap.Spec.CNI.ValuesFrom.ProviderRef.Name, suffixStr)
+		if clusterBootstrap.Spec.CNI.ValuesFrom.ProviderRef.Kind == "AntreaConfig" {
+			name = name + "-antrea"
```

Comment:
need to add another check to catch the. case where the name DOES hae the "-antrea" sufix. 

if clusterBootstrap.Spec.CNI.ValuesFrom.ProviderRef.Kind == "AntreaConfig" && !strings.HasSuffix(name, "-antrea"){
             name = name + "-antrea"
}